### ISSUE TITLE
Failed to send activity: bot returned an error

### ISSUE THREAD

**Issue by AkhilSainath** [microsoft/BotFramework-WebChat#4273]
![image](https://user-images.githubusercontent.com/83917090/166890492-f977325d-1fd7-43e7-98ae-594fc50b87d8.png)

please do help me out

**Comment by lauren-mills** [comment]
HI @AkhilSainath, can you provide any more details around how you are setting up your web chat client code? Also, does your bot work in the Bot Framework Emulator and the Test in Web Chat tab in your Azure settings?

This information will help us better assist you.

**Comment by AkhilSainath** [comment]
Yes, it works in an emulator, I implemented OAuthPrompt to signin. It
worked and i deployed it as well.

But my team wants that prompt to be not appearing, so I tried yours, cross
cheked multiple times. It didn't work.

I followed the same steps in your code and in fact I used your same code
repo... but it didn't work.

Can I please get some of your time to help me out in this (whenever you
can).

Thanks & Regards,

Akhil

**Comment by lauren-mills** [comment]
Are these the sample and instructions you followed? https://github.com/microsoft/BotFramework-WebChat/tree/main/samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication

It's unclear which code you are referencing in your messages.

**Comment by AkhilSainath** [comment]
Yes, i followed these

https://github.com/microsoft/BotFramework-WebChat/tree/main/samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication

Thanks & Regards,

Akhil

**Comment by AkhilSainath** [comment]
Hi Lauren,

Greetings.

Myself and my team is trying to integrate a bot which we developed to the
website, we followed the same
https://github.com/microsoft/BotFramework-WebChat/tree/main/samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication,
we couldn't do it.

even we tried to clone you own repo mentioned in this
https://github.com/microsoft/BotFramework-WebChat/tree/main/samples/07.advanced-web-chat-apps/e.sso-on-behalf-of-authentication
and
we followed the same steps as you said, It didn't work.

Is it possible for you to spend time on this? this weekend. If you want, we
are ready to pay you for the same.

Please let us know your thoughts on this?

Thanks & Regards,

Akhil

**Comment by stevkan** [comment]
@AkhilSainath, are you able to share your web chat code (not including any secrets/keys) in a zip file and post it here? This way, we can diagnose more closely what is happening?

**Comment by khanfaizan123** [comment]
did you guys find any solution of this query?
